packages/docs/src/reusable-text-snippets/ColorContrastGuidelines.mdx

```diff
@@ -0,0 +1,21 @@
+## Accessibility considerations
```

Comment:
i'm not sure where this text component should live, so i'm open to ideas on location and component/dir name as i _think_ this is the first time we're reusing mdx in this way.